root/static/scripts/timeline.js

```diff
@@ -212,13 +212,14 @@ class TimelineViewModel {
 
         _.forEach(parts, function (part) {
             var match;
-            if (match = part.match(/^(-)?([rv])-?$/)) { // trailing - for backwards-compatibility
+
```

Comment:
Super nitpicky, but: this is adding an extra line, is it intentional?

The next line is too long (it already was, not your fault) - maybe it would make sense to move the comment to this line instead?

@mwiencek: opinion? :)

Reply:
Hi, so i should remove the extra line (sorry i have a habit to add a line after variable declaration) and move the comment to the next line right?

Reply:
Hi, yes the extra line was intentional since for me that makes the code just a little more readable. Should i remove it?

Reply:
Let's see what mwiencek says :)

Reply:
Okay :) maybe you and i have a different "code taste"

Reply:
I mean, I don't actually dislike it or anything, just want to make sure we're being consistent.

Reply:
I get it :D

Reply:
I don't have a problem with the extra line. :)